sandwich/src/Test/Sandwich/Types/Spec.hs

```diff
@@ -126,7 +126,7 @@ data FailureReason = Reason { failureCallStack :: Maybe CallStack
                    | RawImage { failureCallStack :: Maybe CallStack
                               , failureFallback :: String
                               , failureRawImage :: Image }
-  deriving (Show, Typeable, Eq)
+  deriving (Show, Eq)
```

Comment:
I know recent GHCs warn that Typeable is auto-derived, but I think maybe we should keep this for compatibility with older GHCs?

Reply:
GHC 7.10 auto deriving `Typeable`.
sandwich support lower is GHC 8.10.
I think no problem.